files/en-us/learn/server-side/django/development_environment/index.html

```diff
@@ -129,9 +129,8 @@ <h3 id="Ubuntu_20.04">Ubuntu 20.04</h3>
 
 <p>Ubuntu Linux 20.04 LTS includes Python 3.8.5 by default. You can confirm this by running the following command in the bash terminal:</p>
 
-<pre class="brush: bash"><span style="line-height: 1.5;">python3 -V
- Python 3.8.5</span>
-</pre>
+<pre class="brush: bash">python3 -V
```

Comment:
Would this work, too?
```suggestion
<pre class="brush: sh">python3 -V
```